fiftyone/utils/eval/detection.py

```diff
@@ -45,6 +85,9 @@ def evaluate_detections(
     dynamic=True,
     custom_metrics=None,
     progress=None,
+    workers=None,
+    batch_method="id",
+    parallelize_method=None,
```

Comment:
Are these the right default values?

Reply:
Yes - batch method "id" is preferable to "slice" because it operates better for a view. Workers=None allows us to read from config.